docs/rep/cli/README.md

```diff
@@ -0,0 +1,81 @@
+# RIK CLI
+
+RIK CLI is a user interface for the rustlang based cloud orchestrator **Rust In Kube**.
+It will send and receive HTTP requests to/from the REST API developed by the controler team.
+
+It will use the crate [clap](https://crates.io/crates/clap), a command line argument parser.
+
+# Cluster connection
+
+RIK CLI will use a `rik.config.yaml` file to establish the connection with the REST API.
+
+TBD : will the API need an authentication token to allow the connection ?
+
+> _rik.config.yaml_ :
+
+```
+cluster:
+  name: cluster name
+  server: API-address
+  auth-token: ApyTUlpd0JnNmRXVGpnaG5BNGhMRXVYVldNSHd1d3gxT0Nac3RmMWpqdEIv
+```
+
+The CLI will parse this file on every command, to get the API address and the auth token.
+Each request will contain the authentication token in the headers, and the YAML file in the payload.
+
+## Commands
+
```

Comment:
`pod` is a `riktl` command. You should list and document all supported commands and then for each command, list and document each subcommand. For example:

## Commands

`riktl` supports the following commands:

* `pod` for managing pods in a RIK cluster.

### `riktl pod`

The `pod` command allows for managing pods through a RIK cluster. It provides sub commands for listing, creating, and deleting pods:

#### `pod list`

bla bla bla

#### `pod create`

foo bar

#### `pod delete`

delete 'em all